modules/ossm-creating-istiocni-resource-using-console.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-istiocni-resource"]
+= Creating the IstioCNI resource using the web console
+:context: ossm-creating-istiocni-resource
+
+The `IstioCNI` resource deploys and configures the {istio} Container Network Interface (CNI) plugin. 
+
+.Prerequisties
+
+
+.Procedure
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Select `istio-cni` in the *Project* drop-down menu.
+
+. Click the Sail Operator.
+
+. Click *IstioCNI*.
+
+. Click *Create IstioCNI*.
+
+. Ensure that the name is `default`.
+
+. Click *Create*. This action deploys the Istio CNI plugin.
++
+When `State: Healthy` appears in the *Status* column, the Istio CNI plugin is successfully deployed.
```

Comment:
In the document, this text appears misaligned with the rest of the content; it is on the same level as the bullet points. Please take a look in the docs

Reply:
That was intentional. It lines up with where the text starts for step 7. It's just describing when things are done.